.github/workflows/trigger-dev-workflow.yaml

```
@@ -0,0 +1,83 @@
+name: Trigger Deploy to AWS Dev
+run-name: "Trigger Deploy - Service: ${{ inputs.service }}, Version: ${{ inputs.version }}, Dev ID: dev${{ inputs.dev_id }}, Config Name: ${{ inputs.config_name }}, Terraform Branch: ${{ inputs.tf_branch }}, Event Type: ${{ inputs.event_type }} to AWS Dev by @${{ github.actor }}"
+
+on:
+  workflow_dispatch:
+    inputs:
+      service:
+        type: string
+        description: "The service you want to deploy. dibbs-ecr-viewer"
+        required: true
+        default: dibbs-ecr-viewer
+      version:
+        type: string
+        description: "The version to deploy. Example: v2.6.0 or main"
+        required: true
+        default: main
+      dev_id:
+        type: choice
+        description: "The unique ID for the dev environment. Must be between 0 and 255."
+        options:
+          - "7"
+          - "42"
+      config_name:
+        type: choice
+        description: "Sets CONFIG_NAME on dibbs-ecr-viewer"
+        required: true
+        options: # not supported yet: AWS_SQLSERVER_NON_INTEGRATED, AWS_SQLSERVER_DUAL
+          - AWS_PG_NON_INTEGRATED
+          - AWS_PG_DUAL
+          - AWS_INTEGRATED
+      tf_branch:
+        type: string
+        description: "The branch to use for the Terraform code. Default is main."
+        required: false
+        default: main
+      event_type:
+        type: choice
+        options:
+          - trigger-dev-plan
+          - trigger-dev-deploy
+        description: "The event type to trigger the workflow."
+        required: true
+
+jobs:
+  trigger_dev_workflow:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Deploy to AWS Dev
+        env:
+          SERVICE: ${{ inputs.service }}
+          VERSION: ${{ inputs.version }}
+          DEV_ID: ${{ inputs.dev_id }}
+          CONFIG_NAME: ${{ inputs.config_name }}
+          TF_BRANCH: ${{ inputs.tf_branch }}
+          EVENT_TYPE: ${{ inputs.event_type }}
+        run: |
+          repo_owner="CDCgov"
+          repo_name="dibbs-aws"
+          service="$SERVICE"
```

I think you can do this pattern instead of the intermediate env vars (if you think more readable/maintainable)
```suggestion
          service="${{ inputs.service}}"
```

---

When passing options into a script, always pass them through the env key for sanitization.

---

ooooh - that's good to know! the places I've done this to date are all hard coded matrices of values, so no security risk